client/reader/stream/x-post.jsx

```diff
@@ -139,10 +139,14 @@ var CrossPost = React.createClass( {
 				<SiteAndAuthorIcon
 					siteId={ this.props.post.site_ID }
 					isExternal={ this.props.post.is_external }
-					user={ post.author } />
+					user={ post.author }
+					onClick={ this.handleTitleClick }
+					href={ post.URL } />
 				<div className="reader__x-post">
 					{ post.title
-						? <h4 className="reader__post-title"><a className="reader__post-title-link" onClick={ this.handleTitleClick } href={ post.URL } target="_blank" rel="noopener noreferrer">{ xpostTitle }</a></h4>
+						? <h4 className="reader__post-title">
```

Comment:
FFTI but in general I think this logic is better written with `&&` instead of the ternary `?`:
```javascript
{ post.title && 
  <h4 .........> 
    ...
  <h4>
}
```